java/src/test/java/com/ncc/aif/ValidateCLITest.java

```diff
@@ -138,6 +138,24 @@ void correctDepthWithoutValue() {
         }
     }
 
+    @Nested
+    class HypothesisMaxSizeArgument {
+        @Test
+        void hypothesisMaxSizeBadType() {
+            expectUsageError(ValidateAIFCli.ERR_BAD_ARGTYPE.replaceAll("%.", ""),
+                    "--ldc", "--hypothesis-max-size", "s", "-t", "2", "-f", "tmp.ttl");
+        }
+        @Test
+        void hypothesisMaxSizeNegative() {
+            expectUsageError(ValidateAIFCli.ERR_BAD_ARGTYPE.replaceAll("%.", ""),
+                    "--ldc", "--hypothesis-max-size", "s", "-10", "2", "-f", "tmp.ttl");
```

Comment:
It looks like you're sending `--hypothesis-max-size` both an `s` and a `-10` as arguments.  I think you just want to send the -10?

Reply:
Fixed.